### PR TITLE
JSON in TOML out

### PR DESCRIPTION
**Pull request by aciidb0mb3r**

### CODE REVIEW

File: Sources/PackageDescription/JSON.swift [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        case .null:
+            return "null"
+        case .bool(let value):
+            return value ? "true" : "false"
+        case .int(let value):
+            return value.description
+        case .double(let value):
+            return value.debugDescription
+        case .string(let value):
+            return "\"" + value + "\""
```
Comment (ddunbar) [inline Sources/PackageDescription/JSON.swift]: We probably need to worry a little about escapes here

Comment (ddunbar) [inline Sources/PackageDescription/JSON.swift]: Oh huh I guess old version didn't...

Comment (czechboy0) [inline Sources/PackageDescription/JSON.swift]: I had the same thought: https://github.com/apple/swift-package-manager/pull/604#discussion_r74685276

**Comment by ddunbar** [comment]
LGTM, thanks!

**Pull request merged**